File: add-to-app/android-app/app/src/main/java/com/github/techisfun/NativeApiImpl.kt
package com.github.techisfun

class NativeApiImpl: PigeonApi.NativeApi {

    override fun getPerson(): PigeonApi.PersonMessageModel {
        val personMessageModel = PigeonApi.PersonMessageModel().apply {
            name = "Pigeon Android"
        }

        return personMessageModel
    }
}

File: add-to-app/ios-app/Flutter Add To App Sample/NativeApiImpl.swift
//
//  NativeApiImpl.swift
//  Flutter Add To App Sample
//
//  Created by Andrea on 22/01/22.
//

import Foundation
import Flutter

class NativeApiImpl: NSObject, NativeApi {
    
    func getPersonWithError(_ error: AutoreleasingUnsafeMutablePointer<FlutterError?>) -> PersonMessageModel? {
        let person = PersonMessageModel()
        person.name = "IOS Pigeon"
        
        return person
    }
    
}
